### DISCUSSION TITLE
RSS feed does not include media:content tag

### DISCUSSION THREAD

**Discussion by rohitfarmer** [quarto-dev/quarto-cli#6930] [Q&A] [answered]
### Description

Hi, I am trying to figure out how to include <media:content>  tag in the RSS feed generated by Quarto. Is there a way to generate this tag using the image tagged as `.preview-image`? It's what Pinterest looks for in auto-generating pins from RSS feeds. https://help.pinterest.com/en/business/article/auto-publish-pins-from-your-rss-feed

Here is the RSS feed from my website [https://outdoorsyindians.com/posts.xml](https://outdoorsyindians.com/posts.xml)

Any suggestions would be helpful. Thanks!

**Comment by dragonstyle** [comment, marked as answer]
We currently won't generate this tag for an automatically detected preview image - we instead will only include the `media-content` tag when there is an `image` specified in the front matter for the document:

```markdown
---
title: My Document
author: Norah Jones
image: preview.png
---
```

I'll take a quick look at adding this for automatically discovered preview images, but this is a workaround you could use right now...

**Reply by rohitfarmer** [reply]
Thank you for your response. Yes, a native feature to pull .preview-image would be great. I tried this workaround, and it worked. I have another related question. If I use "image:" in the front matter and want to pull the same image as the blog post banner/featured/preview image in the post. How can I do it? I tried as below. The variable pulls the image path, but it adds a ".png" extension in the end ("https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg.png"), altering the URL. How can I fix it? Or a better method to include the header image in a non-repetitive and cleaner way.


```
---
title: "Essentials for Outdoor Survival"
pagetitle: "{{< meta title >}}"
image: https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg
image-alt: "Photo of a bushcraft shelter"
---

![{{< meta image-alt >}}]({{< meta image >}})

```

**Reply by dragonstyle** [reply]
I'm not able to reproduce the `png` suffix issue in Quarto 1.4.376 - the follow appears to work just fine:

```markdown
---
title: My Title
image: https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg
---


![My Image]({{< meta image >}})
```

Any you can update to the pre-release?

**Reply by rohitfarmer** [reply]
I am on version 1.3.450. I tried your code, and it's the same. I changed the image URL as well.

**Reply by rohitfarmer** [reply]
I upgraded to the pre-release version and it worked
